frontend/workflows/experimentation/src/list-view.tsx

```diff
@@ -158,7 +158,7 @@ const ListView: React.FC<ListViewProps> = ({ columns, items, onRowSelection }) =
                     return (
                       <TableRow
                         hover
-                        onClick={event => handleClick(event, item)}
+                        onSelect={event => handleClick(event, item)}
```

Comment:
```suggestion
```

Reply:
You want to move `onClick={event => handleClick(event, item)}` onto the TableCell

Reply:
Mhm - that wouldn't feel native I guess, people usually except to be able to follow through with one click 😞

Reply:
yeah unfortunately I'm not sure it's super easy to know that a user is intending to click and navigate or click and highlight